pages/product.js

```diff
@@ -120,70 +138,99 @@ class ProductPage extends React.PureComponent{
         `}</style>
         <div className="breadcrumb">
           <a href="/">Home</a>
-          {router.asPath.replace(/%20/g,' ')}
+          {router.asPath.replace(/%20/g, ' ')}
         </div>
         <div className="product_container">
```

Comment:
Can you move out small small components from this along with their CSS? The page is too long to maintain.

Reply:
I am working on it @bogas04. Will push the change by tonight.
@Nshul There will be merge conflicts in merging this branch since I have already started working on  the other branch. I will try to resolve them if possible.